store/storetest/mocks/UserStore.go

```diff
@@ -848,3 +848,35 @@ func (_m *UserStore) VerifyEmail(userId string) store.StoreChannel {
 
 	return r0
 }
+
+// GetUserInteractions provides a mock function with given fields: uid, teamId, learningGroupPrefixes
+func (_m *UserStore) GetUserInteractions(uid string, teamId string, learningGroupPrefixes []*model.LearningGroupPrefix) store.StoreChannel {
+	ret := _m.Called()
+
+	var r0 store.StoreChannel
+	if rf, ok := ret.Get(0).(func() store.StoreChannel); ok {
+		r0 = rf()
+	} else {
```

Comment:
I think here you can use `else if {` instead of `else { if {`.

Reply:
I agree, but this syntax is all over the file. I vote to keep it as is.

Reply:
Good point, I keep missing the local conventions.